src/components/language-switch/language-switch.tsx

```diff
@@ -0,0 +1,47 @@
+import { useI18next } from 'gatsby-plugin-react-i18next';
+import React from 'react';
+import styled from 'styled-components';
+import { Link as LanguageLink } from 'gatsby-plugin-react-i18next/dist/Link';
+
+const StyledLanguageLink = styled(LanguageLink)<{ selected?: boolean }>`
```

Comment:
I'm not so happy about the usage of `LanugaLink`. We should use our common `Link` component everywhere to a central place to handle links. With this component here we create an exception - we now have 2 places where links are rendered.

Links need to be treated all in the same way, regardless of where they appear. If we have 2 places where we create links, we always need to keep them in sync. 

Is it somehow possible to use our own `Link` component here?